pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1220,6 +1227,71 @@ func isUSBNeeded(c *ConverterContext, vmi *v1.VirtualMachineInstance) bool {
 	return false
 }
 
+func setupDomainMemory(c *ConverterContext, vmi *v1.VirtualMachineInstance, domain *api.Domain) error {
+
+	if vmi.Spec.Domain.Memory == nil || vmi.Spec.Domain.Memory.MaxGuest == nil {
+		var err error
+
+		domain.Spec.Memory, err = vcpu.QuantityToByte(*vcpu.GetVirtualMemory(vmi))
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	maxMemory, err := vcpu.QuantityToByte(*vmi.Spec.Domain.Memory.MaxGuest)
+	if err != nil {
+		return err
+	}
+
+	domain.Spec.MaxMemory = &api.MaxMemory{
+		Unit:  maxMemory.Unit,
+		Value: maxMemory.Value,
+		Slots: 1,
+	}
+
+	domain.Spec.Memory = maxMemory
+
+	initialMemoryApi, err := vcpu.QuantityToByte(*vmi.Spec.Domain.Memory.Guest)
+	if err != nil {
+		return err
+	}
+
+	if domain.Spec.CPU.NUMA == nil || len(domain.Spec.CPU.NUMA.Cells) == 0 {
+		domain.Spec.CPU.NUMA = &api.NUMA{
+			Cells: []api.NUMACell{
+				{
+					ID:     "0",
+					CPUs:   fmt.Sprintf("0-%d", domain.Spec.VCPU.CPUs-1),
+					Memory: initialMemoryApi.Value,
+					Unit:   initialMemoryApi.Unit,
+				},
+			},
+		}
+	} else {
+		domain.Spec.CPU.NUMA.Cells[0].Memory = initialMemoryApi.Value
+		domain.Spec.CPU.NUMA.Cells[0].Unit = initialMemoryApi.Unit
+	}
```

Comment:
I am not quite sure this is what we want. Could you elaborate here? Afaik this cannot be supported at the moment.

Reply:
That's just a mistake, removing it.